examples/lusid/fx-forwards/create-fx-forward.sql

```diff
@@ -0,0 +1,30 @@
+-- ===============================================================
+-- Description:
+-- In this query, we add an Fx Forward holding in to LUSID.
+-- ===============================================================
+
+-- Defining scope variable
+@@instrumentScope = select 'luminesce-examples';
```

Comment:
I think Stephen wanted a space between these comments and the code section, could you add it and for the below sections too?

Reply:
Newline spacing added to both files